packages/indexers/src/unruggableMemecoin-transfers.indexer.ts

```diff
@@ -0,0 +1,70 @@
+import { Block, hash } from "./deps.ts";
+import { FACTORY_ADDRESS, STARTING_BLOCK } from "./unruggableMemecoin.ts";
+
+export const config = {
+  filter: {
+    header: { weak: true },
+    events: [
+      {
+        fromAddress: FACTORY_ADDRESS,
+        keys: [hash.getSelectorFromName("MemecoinLaunched")],
+        includeReceipt: false,
+      },
+    ],
+  },
+  streamUrl: "https://mainnet.starknet.a5a.ch",
+  startingBlock: STARTING_BLOCK,
+  network: "starknet",
+  finality: "DATA_STATUS_ACCEPTED",
+  sinkType: "postgres",
+  sinkOptions: {
+    connectionString: "",
+    tableName: "unrugmeme_transfers",
+  },
+};
+
+export function factory({ header, events }) {
+  const launchEvents = (events ?? []).map(({ event }) => {
+    const memecoin_address = event.data[0];
+    return {
+      fromAddress: memecoin_address,
+      keys: [hash.getSelectorFromName("Transfer")],
+      includeReceipt: false,
+    };
+  });
+
+  return {
+    filter: {
+      header: { weak: true },
+      events: launchEvents,
+    },
+  };
+}
+
+
+export default function DecodeUnruggableMemecoinLaunch({ header, events }: Block) {
+  const { blockNumber, blockHash, timestamp } = header!;
+
+  return (events ?? []).map(({ event, transaction }) => {
+    const transactionHash = transaction.meta.hash;
+    const transferId = `${transactionHash}_${event.index ?? 0}`;
+    const fromAddress = event.keys[1];
+    const toAddress = event.keys[2];
+    const amount = BigInt(event.data[0]);
```

Comment:
Since amount is a `uint256` , you should parse the full amount (event tho it's very unlikely it will ever be used).

```suggestion
    const amount = uint256.uint256ToBN({ low: event.data[0], high: event.data[1] })
```